src/App.stories.tsx

```diff
@@ -1,6 +1,7 @@
-import React from 'react';
+/** @jsxRuntime classic */
+/** @jsx jsx */
```

Comment:
We should only need the pragma if we're [not using the `babel-preset-css-prop` plugin](https://emotion.sh/docs/css-prop#jsx-pragma)

Do you know why we need both here?